diesel/src/query_dsl/load_dsl.rs

```diff
@@ -25,6 +25,9 @@ where
     for<'a> Self: LoadQueryGatWorkaround<'a, 'query, Conn, U>,
 {
     /// Load this query
+    #[diesel_derives::__diesel_public_if(
+        feature = "i-implement-a-third-party-backend-and-opt-into-breaking-changes"
```

Comment:
do we want to even make the whole trait private?

Reply:
The trait itself is required as bound for generic code, so that users can call `RunQueryDsl::load` (and `load_iter`) in a generic context.